### DISCUSSION TITLE
Use the same MX as Proton due to blocking

### DISCUSSION THREAD

**Discussion by lucasmchoi** [simple-login/app#1550] [Feature Requests]
Since on some sites they are blocking even custom domains used with simple login by checking the mx records it would be nice to use the same mx records as proton since your joined them. This would possibly resolve issues with blocking.

**Comment by hanseinotabi** [comment]
Does this happen a lot even with custom domains? I'm trying to sort out my email strategy going forward and I was considering using Simple Login to create addresses for my custom domains, but if there's a chance I won't get emails for professional use, maybe this isn't such a good idea?

**Comment by Coder84619** [comment]
Which sites are going as far as checking MX records for a custom domain?

**Reply by Ambiguity4142** [reply]
I do believe that IceDrive is doing so. There are a couple others that I cannot remember off the top of my head.

**Comment by lucasmchoi** [comment]
A possibility would be to provide the proton mx servers only for paying customers and maybe do a more restrictive scan for spam before sending.

**Comment by lllIIIlll** [comment]
Yes, please. I just found my first site blocking my custom domain by checking the MX records.
In case you're wondering, it was https://www.synology.com

**Reply by nguyenkims** [reply]
We have reached out to this website to solve this issue.

**Reply by lllIIIlll** [reply]
Many thanks. I found one more site that blocks my custom domain and other simplelogin addresses.
It's a Korean site https://ridibooks.com/ebook
Direct support email address [help@ridi.com](mailto:help@ridi.com)

**Comment by jnv** [comment]
Another site which deliberately blocks SimpleLogin's MX records is Netlify: https://answers.netlify.com/t/email-is-invalid/85970

**Reply by nguyenkims** [reply]
Thanks, we'll reach out to netlify to solve this issue.

**Comment by xyxzxyz** [comment]
Github also restrict accounts that use email addresses point to simplelogin servers. I changed my Github email address to one custom domain that points to simplelogin servers, Github restrictedmy account. Then I have to appeal, changed my email address to outlook.com to have my account unrestricted.
